src/main/java/net/dries007/tfc/common/blockentities/PitKilnBlockEntity.java

```diff
@@ -274,6 +276,14 @@ public NonNullList<ItemStack> getStraws()
         return strawItems;
     }
 
+    public Long getLogsCount() {
+        return logItems.stream().filter(Predicate.not(ItemStack::isEmpty)).count();
+    }
+
+    public Long getStrawsCount() {
+        return strawItems.stream().filter(Predicate.not(ItemStack::isEmpty)).count();
+    }
```

Comment:
```suggestion
    public int getLogCount()
    {
        return (int) logItems.stream().filter(Predicate.not(ItemStack::isEmpty)).count().orElse(0);
    }

    public int getStrawCount()
    {
        return (int) strawItems.stream().filter(Predicate.not(ItemStack::isEmpty)).count().orElse(0);
    }
```

- Proper English (`getLogsCount` -> `getLogCount`)
- Use our code style (braces on newlines)
- Don't make this return a nullable `Long` which is then silently unboxed later - it's never intended to be zero, return an `int`, and perform the safe unboxing here.